integrations/livepeer/components/stream.tsx

```diff
@@ -0,0 +1,49 @@
+'use client'
+
+import { useMemo, useState } from 'react'
+
+import { Player, useCreateStream } from '@livepeer/react'
+
+import { Button } from '@/components/ui/button'
+import { Input } from '@/components/ui/input'
+
+export const Stream = () => {
+  const [streamName, setStreamName] = useState<string>('')
+  const { mutate: createStream, data: stream, status } = useCreateStream(streamName ? { name: streamName } : null)
+
+  const isLoading = useMemo(() => status === 'loading', [status])
```

Comment:
there's no need for `useMemo` here
```suggestion
  const isLoading = status === 'loading'
```